sphinx/source/docs/user_guide/annotations.rst

```diff
@@ -1,70 +1,84 @@
-any Chart function:
+If you use the :ref:`bokeh.plotting <userguide_interfaces_plotting>` interface,
+the quickest way to add a basic title is to pass the text as the ``title``
+parameter to |Figure|:
 
 .. bokeh-plot:: docs/user_guide/examples/plotting_title_basic.py
     :source-position: above
 
-The default title is normally on the top of a plot, aligned to the left. The
-``title_location`` parameter controls on which side of the plot the title appears:
+The default title is generally located above a plot, aligned to the left.
+
+To define the placement of the title in relation to the plot, use the
+``title_location`` parameter. A title can be located above, below, left, or
+right of a plot. For example:
 
 .. bokeh-plot:: docs/user_guide/examples/plotting_title_location.py
     :source-position: above
 
-The default |Title| is accessible through the ``Plot.title`` property.
-Visual properties for font, border, background, and others can be set
-directly on ``.title``. Here is an example that sets font and background
-properties as well as the title text and title alignment using ``.title``:
+Use your plot's ``.title`` property to customize the default |Title|. You can
+set visual properties for font, border, and background, for example.
+
+This example uses the ``.title`` property to set the font and background
+properties as well as the title text and title alignment:
 
 .. bokeh-plot:: docs/user_guide/examples/plotting_title_visuals.py
     :source-position: above
 
-Note that the alignment is measured along the direction of text. For
-example, for titles on the left side of a plot, "left" will be in the
-lower corner.
+Note that the ``align`` property is relative to the direction of the text. For
+example: If you have placed your title on the left side of your plot, setting
+the ``align`` property to ``"left"`` means your text is rendered in the lower
+left corner.
 
-In addition to the default title, you can create and add
-additional |Title| objects to plots using the ``add_layout`` method
-of Plots:
+To add more titles to your document, you need to create additional |Title|
+objects. Use the :func:`~bokeh.models.plots.Plot.add_layout` method of your plot
+to include those additional |Title| objects in your document:
 
 .. bokeh-plot:: docs/user_guide/examples/plotting_title_additional.py
     :source-position: above
 
-If a title and a sticky toolbar are set to the same side, they will occupy
-the same space:
+If a title and a :ref:`toolbar <userguide_tools>` are placed on the same side
+of a plot, they will occupy the same space:
 
 .. bokeh-plot:: docs/user_guide/examples/plotting_title_toolbar.py
     :source-position: above
 
 If the plot size is large enough, this can result in a more compact plot.
 However, if the plot size is not large enough, the title and toolbar may
-visually overlap in a way that is not desirable.
+visually overlap and become impossible to read.
```

Comment:
being pedantic, it may still be possible to read, only be difficult or just look very bad. Maybe just leave it at "may visually overlap".